doc/changes/latest.inc

```diff
@@ -37,6 +37,8 @@ Bugs
 
 - Fix bug where ``theme`` was not handled properly in :meth:`mne.io.Raw.plot` (:gh:`10487`, :gh:`10500` by `Mathieu Scheltienne`_ and `Eric Larson`_)
 
+- Fix bug with :meth:`raw.crop(start, stop) <mne.io.Raw.crop>` when ``raw.info['meas_date'] is None`` where annotations would shift when ``start != 0`` (:gh:`10491` by `Eric Larson`_)
```

Comment:
I had trouble understanding what you were saying, so I tried to rephrase is such that it be easier for my brains to digest!
```suggestion
- Fix bug in :meth:`raw.crop(start, stop) <mne.io.Raw.crop>` that would cause annotations to be erroneously shifted when ``start != 0`` and no measurement date was set. (:gh:`10491` by `Eric Larson`_)
```